src/main/java/dao/CommonDao.java

```diff
@@ -0,0 +1,45 @@
+package dao;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
```

Comment:
Do we need to use both ```lombok``` import?

Reply:
yes, because '*' loads classes, but not internal packages like experimental